### ISSUE TITLE
Enhance mail-replied icon contrast

### ISSUE THREAD

**Issue by MaxSchambach** [numixproject/numix-icon-theme#1274]
## Issue
The `actions/mail-replied.svg` icons use a slight green color for the "replied arrow", in accordance to the reply and group reply icons which are also green.
Unfortunatly, the `mail-replied.svg` icon usually is displayed in very small sizes in an email application. Due to the very low contrast of the green on white/blue, the mail-replied icon is not really distinguishable from the regular mail icon.

## Suggestion:
Changing the color of the replied arrow of the `actions/mail-replied.svg` icon to red in order to increase contrast. Yes, this clashes with the color of the reply and group reply arrow, but I think it is worth it, as it is something you look at very often working with emails and you cannot really tell which ones you've replied to by just looking at the icon.

[mail-replied.pdf](https://github.com/numixproject/numix-icon-theme/files/1505486/mail-replied.pdf)
(I have attached the icon proposal as .pdf as upoading .svg is not supported, I could also prepare it for a PR)

**Comment by wa4557** [comment]
I guess using a different kind of green would be the way to go. Using red makes no sense here

**Comment by MaxSchambach** [comment]
I disagree. Green on blue/white will always be a weak contrast, making it hard to read when the icon is really tiny. Red has the most contrast in this, and even though it is not conform with the "reply" button's green color, it is a color that is repeatedly used in the theme.
As the "send" arrow in the "send/recieve" botton is also red, it also kind of makes sense to make the sent arrow of the mail red. One could therefore think about making the Reply and Group Reply button red as well.

Also, one could also change the Outbox' arrow color to red (which is currently yellow)
